resources/i18n/fr.json

```diff
@@ -148,9 +148,9 @@
                 "name": "Nom",
                 "duration": "Durée",
                 "ownerName": "Propriétaire",
-                "public": "Public",
+                "public": "Publique",
```

Comment:
[Playlist](https://www.larousse.fr/dictionnaires/francais/playlist/10910433) is feminine in French. The `updateAt` field is already using the feminine inflexion ("mis**e**").

I did the same for `public` and `createdAt` fields.